components/mdx-components.tsx

```diff
@@ -91,14 +91,7 @@ const components = {
       {...props}
     />
   ),
-  img: ({
-    className,
-    alt,
-    ...props
-  }: React.ImgHTMLAttributes<HTMLImageElement>) => (
-    // eslint-disable-next-line @next/next/no-img-element
-    <img className={cn("rounded-md border", className)} alt={alt} {...props} />
-  ),
+  Image: (props: ImageProps) => <Image {...props} />,
```

Comment:
should we add css and classname as well?

Reply:
no